tools/AzureRM/AzureRM.psd1

```diff
@@ -103,7 +103,8 @@ RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '3.4.1';
                @{ModuleName = 'AzureRM.Tags'; RequiredVersion = '3.4.1'; }, 
                @{ModuleName = 'AzureRM.TrafficManager'; RequiredVersion = '3.4.1'; }, 
                @{ModuleName = 'AzureRM.UsageAggregates'; RequiredVersion = '3.4.1'; }, 
-               @{ModuleName = 'AzureRM.Websites'; RequiredVersion = '3.4.1'; })
+               @{ModuleName = 'AzureRM.Websites'; RequiredVersion = '3.4.1'; }),
+			   @{ModuleName = 'AzureRM.ApplicationInsights'; RequiredVersion = '0.1.0'; })
```

Comment:
@frankguodongchen please move this entry so the modules are in alphabetical order for consistency

Reply:
Updated. thanks!